pkg/asset/manifests/vsphere/cloudproviderconfig.go

```diff
@@ -0,0 +1,41 @@
+package vsphere
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/types"
+)
+
+const configTemplate = `[Global]
```

Comment:
nit: is it possible to use `gopkg.in/ini.v1` to created this in a more structured manned...?

Reply:
I'll look at that. I was not aware of that package.